File: Python/tests/test_ug_delta.py
import pytest

from hbayesdm.models import ug_delta


def test_ug_delta():
    _ = ug_delta(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_ug_delta.R
context("Test ug_delta")
library(hBayesDM)

test_that("Test ug_delta", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(ug_delta(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
